test/Sentry.Tests/Platforms/iOS/NativeSerializationTests.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Foundation;
+using Sentry.Cocoa.Extensions;
+using Xunit;
+
+namespace Sentry.Tests.Platforms.iOS;
+
+public class NativeSerializationTests
+{
+    [Fact]
+    public void Managed_To_Native()
```

Comment:
```suggestion
    public void CopyToCocoaSentryEvent_CopiesProperties()
```

See contributor docs on [naming tests](https://github.com/getsentry/sentry-dotnet/blob/main/CONTRIBUTING.md#naming-tests)... just a convention for this repo basically.  I'm not sure exactly what you want to call this test - the above is just a suggestion/idea.